firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -96,10 +96,14 @@
 
   private static final String ETAG = "ETag";
 
-  private static final String INSTANCE_ID_STRING = "fake instance id";
-  private static final String INSTANCE_ID_TOKEN_STRING = "fake instance id token";
-  private static final InstanceIdResult INSTANCE_ID_RESULT =
-      new FakeInstanceIdResult(INSTANCE_ID_STRING, INSTANCE_ID_TOKEN_STRING);
+  private static final String INSTALLATION_ID = "fake instance id";
+  private static final String INSTALLATION_TOKEN = "fake instance id token";
```

Comment:
Please use values that have a roughly correct format in order to make it easier for readers to understand what data is used / required here.
Examples:
FID: "`'fL71_VyL3uo9jNMWu1L60S`"
auth token: "`eyJhbGciOiJF.eyJmaWQiOiJmaXMt.AB2LPV8wRQIhAPs4NvEgA3uhubH`"